kuwala/core/canvas/src/data/state/models/CanvasModel.js

```diff
@@ -91,17 +94,32 @@ const CanvasModel = {
                     source,
                     target,
                     updateDataBlock,
-                    updateTransformationBlock
+                    updateTransformationBlock,
+                    updateExportBlock
                 });
                 addElement(edgeToAdd);
             } else {
                 alert('Maximum number of connections reached!');
             }
+        } else if ( targetElement && targetElement.type === EXPORT_BLOCK) {
```

Comment:
The if conditions should be combined as the executed functions inside the blocks is the same.